.github/workflows/quickstart-checker.yml

```diff
@@ -43,3 +43,37 @@ jobs:
 
       - name: Test quickstart
         run: make style.quickstart
+  test_go_1_17:
+    name: "🍀 Quickstart Validation - go 1.17"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.8
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Cache dependencies
+        uses: actions/cache@v2.1.4
+        with:
+          # Cache
+          path: ~/go/pkg/mod
+          # Cache key
+          key: ubuntu-latest-go-${{ hashFiles('**/go.sum') }}
+          # An ordered list of keys to use for restoring the cache if no cache hit occurred for key
+          restore-keys: |
+            ubuntu-latest-go-
+
+      - name: Get dependencies
+        run: |
+          if [ -f Gopkg.toml ]; then
+              curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+              dep ensure
+          else
+              go get -v -t -d ./...
+          fi
+
+      - name: Test quickstart
+        run: make style.quickstart.17
```

Comment:
I wonder why separating quickstart check from two different go versions?
make style.quickstart.17
make style.quickstart

Can we merge two jobs into one? Any concerns?

Reply:
Since [the dubbo quickstart](https://mosn.io/layotto/#/en/start/rpc/dubbo_json_rpc) can only be run with go 1.17 , I add this workflow for it
![image](https://user-images.githubusercontent.com/26001097/167302148-0714fd88-d012-42db-b025-b235ce6be989.png)

Actually the only difference between `style.quickstart.17` and `style.quickstart` is that we use different `quickstarts` variables:
![image](https://user-images.githubusercontent.com/26001097/167302111-caabb892-9418-4b95-a593-7b335f07a849.png)
I think the two workflows can reuse same shell script but pass different `quickstarts` variables. But I don't know how to write it 🤔 
Any suggestions on it?

Reply:
see https://github.com/seeflood/layotto/pull/6

Reply:
we do not need to start another shell to do this, just pass a value from path to determine what jobs should we run.

Reply:
@Xunzhuo Cooool!  Shell script Master !